packages/bpk-component-graphic-promo/src/BpkGraphicPromo.module.scss

```
@@ -0,0 +1,256 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+@import './custom-breakpoints.module';
+
+.bpk-graphic-promo {
+  max-width: 65.5rem;
+  margin-right: auto;
+  margin-left: auto;
+  background-color: $bpk-color-sky-blue;
+  background-position: center;
+  background-size: cover;
+  box-shadow: none;
+
+  @include breakpoint-landscape {
+    pointer-events: none;
+
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-small();
```

Within Backpack we don't have a spacing value of `bpk-spacing-small()` but rather `bpi-spacing-sm()`

```suggestion
      margin-bottom: bpk-spacing-sm();
```

---

Sorted. 🙂